sycl/test-e2e/Matrix/joint_matrix_copy_impl.hpp

```diff
@@ -0,0 +1,147 @@
+#define TM 8
```

Comment:
please take a look at common.hpp
When adding new tests, we should use common things from there, also no use of macros (see joint_matrix_out_bounds for an example)

Reply:
We already have similar test: https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Matrix/joint_matrix_down_convert_impl.hpp.
I suggest instead of introducing new test which is just copy of several other tests, just modify "https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Matrix/joint_matrix_down_convert_impl.hpp" to use new API.
Also we don't need to call MAD to test joint_matrix_copy API, so this part is unnecessary IMO.